python/ray/dashboard/modules/job/job_manager.py

```diff
@@ -263,6 +263,7 @@ async def _monitor_job_internal(
                 num_consecutive_failures = 0
 
             except Exception as e:
+                job_status = await self._job_info_client.get_status(job_id)
```

Comment:
This line as accidentally removed from the previous PR.

Reply:
this must have been a mistake during a merge. Thanks for fixing

Reply:
Oh, this doesn't work since we want to retry this RPC in case of transient network error.